common/views/components/InfoBox/InfoBox.js

```diff
@@ -20,43 +20,42 @@ const InfoBox = ({
   items,
   children
 }: Props) => (
-  <div className={classNames([
-    'bg-yellow',
-    spacing({s: 4}, {padding: ['top', 'right', 'bottom', 'left']}),
-    spacing({s: 4}, {margin: ['top', 'bottom']})
-  ])}>
+  <Fragment>
     <h2 className='h2'>{title}</h2>
-
-    {items.map(({title, description, icon}) =>
-      <Fragment key={title}>
-        {!icon && <h3 className={font({s: 'HNM4'})}>{title}</h3>}
-        {icon &&
-          <h3 className={classNames([
-            'no-margin flex flex--v-center',
-            font({s: 'HNM4'})
-          ])}>
-            <span className={classNames([
-              'flex flex--v-center',
-              spacing({s: 1}, {margin: ['right']})
-            ])}>
-              <Icon name={icon} />
-            </span>
-            <span>{title}</span>
-          </h3>}
-        {description &&
-          <div className={classNames([
-            'plain-text',
-            font({s: 'HNL4'}),
-            spacing({s: 4}, {margin: ['bottom']})
-          ])}>
-            <PrismicHtmlBlock html={description} />
+    <div className={classNames([
+      'bg-yellow',
+      spacing({s: 4}, {padding: ['top', 'right', 'bottom', 'left']}),
+      spacing({s: 4}, {margin: ['bottom']})
```

Comment:
@davidpmccormick what do you think?

I allowed `classNames` to take arrays OR objects for backwards compatibility, but thought we would have 1 way to rule them all (objects)?